petclinic.yaml

```diff
@@ -11,16 +11,21 @@ tools:
     type: dockerImage
     image: maven:3.5.4-jdk-8
     entryPoint: sleep infinity
-    persistentVolumes:
+    volumes:
       - name: maven-repo
-        containerPath: /root/.m2
-    services:
+        mountPath: /root/.m2
```

Comment:
hello, what is the meaning of mountPath ? 
For a volume it's always something `mounted` no ?

How is handled subpath BTW ?

Reply:
it is path where volume folder will be mounted inside of container

Reply:
in k8s I have the ability to specify subpath:
```
volumeMounts:
      - mountPath: /var/lib/mysql
        name: site-data
        subPath: mysql
```

Reply:
@benoitf Sorry, I missed your third question. 
It's right. `subPath` feature was not implemented. If it is important on this stage we can register an issue for add `subPath` feature to Che Volumes, now it has only name(key of map where it is stored) and mountPath (which is called `path` there) https://github.com/eclipse/che/blob/07263f1e30089689d71b057f747a44a29283e3c4/core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/Volume.java#L19

Does 
> it is path where volume folder will be mounted inside of container

answer your questions

> what is the meaning of mountPath ?
> For a volume it's always something mounted no ?

?

Reply:
thx @sleshchenko 
maybe we could agree on the model first and then see which optional field would be or not implemented ?

Reply:
> maybe we could agree on the model first and then see which optional field would be or not implemented ?

Sure. If we are talking about model I think it makes sense to consider adding subpath to `CheVolume`, `che-plugin.yaml` and `Devfile`.

Reply:
If we have subpath how we would implement that ono OSIO when subpaths already used for separating workspaces, volumes of workspaces. Would we also use the third level of nesting?